modules/translation/translation.go

```diff
@@ -118,7 +127,16 @@ func (l *locale) Language() string {
 
 // Tr translates content to target language.
 func (l *locale) Tr(format string, args ...interface{}) string {
-	return i18n.Tr(l.Lang, format, args...)
+	if setting.IsProd {
+		return i18n.Tr(l.Lang, format, args...)
+	}
+
+	// in development, we should show an error if a translation key is missing
+	s, ok := TryTr(l.Lang, format, args...)
+	if !ok {
+		log.Error("untranslated i18n translation key: %q", format)
```

Comment:
```suggestion
		log.Error("missing i18n translation key: %q", format)
```
or something different but the two "translat..." are too much